<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteProjectSyncDialog</class>
 <widget class="QDialog" name="remoteProjectSyncDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>load a remote project</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>730</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_overall">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>750</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_overall">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>750</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading list of remote projects...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_done">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>423</y>
     <width>500</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Successfully loaded all Content</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>751</width>
     <height>371</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::DoubleClicked</set>
   </property>
   <column>
    <property name="text">
     <string>Telescope</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Target</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Change</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>remoteProjectSyncDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>remoteProjectSyncDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
